frontend/src/metabase/query_builder/containers/use-register-query-builder-metabot-context.tsx

```diff
@@ -185,17 +185,15 @@ const getChartConfigs = async ({
   timelines: Timeline[];
 }): Promise<MetabotChartConfig[]> => {
   try {
-    if (!PLUGIN_AI_ENTITY_ANALYSIS.canAnalyzeQuestion(question)) {
-      return [];
-    }
```

Comment:
@sloansparger I figured out that these lines would block us from using this part of the context for something else than entity analysis (e.g. the chart editing at hand). For the button to be available, I saw that we have other checks in place. Let me know whether you can spot potential issues that I haven't accounted for yet when removing this check.

Reply:
This context gets added in the background of every request if you're viewing a question, so you would have this feature by typing "Analyze this chart" into the chat, rather than clicking the analyze chart button, then you'll get the same behavior.

That being said... I think we should just replace the entity analysis feature check with the metabot one. I really don't see the point of gatekeeping a feature as simple as thing 🤷‍♂️